src/csharp/Microsoft.Spark/Broadcast.cs

```diff
@@ -98,8 +98,7 @@ internal void OnSerialized(StreamingContext context)
         /// <returns>Absolute filepath of the created random file</returns>
         private string CreateTempFilePath(SparkConf conf)
         {
-            IJvmBridge jvm = ((IJvmObjectReferenceProvider)conf).Reference.Jvm;
-            var localDir = (string)jvm.CallStaticJavaMethod(
+            var localDir = (string)conf.Reference.Jvm.CallStaticJavaMethod(
```

Comment:
we can just use `Reference` instead of `conf.Reference`?

Reply:
or this is due to the fact that `_jvmObject` is handled differently for this class?

Reply:
We'd have to change the order of the calls in the Constructor, since _jvmObject isn't set yet when this method is called.

Reply:
ok, let's leave this class alone.